internal/operator-controller/applier/boxcutter_test.go

```diff
@@ -81,7 +85,88 @@ func Test_RegistryV1BundleRenderer_Render_Failure(t *testing.T) {
 	require.Contains(t, err.Error(), "some-error")
 }
 
-func Test_SimpleRevisionGenerator_Success(t *testing.T) {
+func Test_SimpleRevisionGenerator_GenerateRevisionFromHelmRelease(t *testing.T) {
```

Comment:
separating the Objects -> Revision, Bundle -> Revision and Chart -> Revision + composition (e.g. Objects -> Revision being used in Chart -> Revision) might also facilitate testing here since we'd only need to test the integration between the chart -> revision and objects -> revision (i.e. that we extract the correct objects from the chart and pass it down). This could help save us time later on as we iterate over the objects to revision process.